.github/workflows/publish.yml

```diff
@@ -37,10 +53,16 @@ jobs:
           certutil -decode certificate/tempCert.txt certificate/certificate.pfx
           Remove-Item –path certificate -include tempCert.txt
           Import-PfxCertificate -FilePath certificate/certificate.pfx -CertStoreLocation Cert:\CurrentUser\My -Password (ConvertTo-SecureString -String $env:WINDOWS_PFX_PASSWORD -Force -AsPlainText)
+
       - name: install app dependencies and build it
         run: yarn && yarn build
-      - uses: tauri-apps/tauri-action@v0
         env:
+          MACOSX_DEPLOYMENT_TARGET: "10.13"
```

Comment:
Why is this necessary?

Reply:
We had this in the CI-tests, so I thought it is probably helpful to have. Don't have strong background on the necessity. 
I remember we were doing some trial/error due to macos version compatibility. Probably a remnant of that.

Do you want us to get rid of this?

Reply:
Both in tests and here. Or explain why it is necessary with link to upstream issue or whatnot.